Jenkinsfile_CNP

```diff
@@ -6,7 +6,81 @@ def type = "java"
 def product = "ccd"
 def component = "message-publisher"
 
+// Variables to switch pipeline logic and wiring per type of build
+def definitionStoreDevelopPr = "PR-575" // This doesn't change frequently, but when it does, only change this value.
+
+// vars needed for functional tests
+// Assume a feature build branched off 'develop', with dependencies develop-to-develop.
+env.DEFINITION_STORE_HOST = "http://ccd-definition-store-api-${definitionStoreDevelopPr}.service.core-compute-preview.internal".toLowerCase()
+
+// Other env variables needed for BEFTA.
+env.IDAM_URL = "https://idam-api.aat.platform.hmcts.net"
+env.S2S_URL = "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
+env.BEFTA_S2S_CLIENT_ID = "ccd_gw"
+env.BEFTA_RESPONSE_HEADER_CHECK_POLICY="JUST_WARN"
+env.OAUTH2_CLIENT_ID = "ccd_gateway"
+env.OAUTH2_REDIRECT_URI = "https://www-ccd.aat.platform.hmcts.net/oauth2redirect"
+env.DM_STORE_BASE_URL = "http://dm-store-aat.service.core-compute-aat.internal"
+env.RD_PROFESSIONAL_API_BASE_URL = "http://rd-professional-api-aat.service.core-compute-aat.internal"
+
+def secrets = [
```

Comment:
Can we delete any secrets not currently required to keep it clean?

Reply:
done